<launch> 
  <node pkg="mocap_optitrack"
    type="mocap_node"
    name="mocap_node"
    respawn="false"
    launch-prefix=""
    required="true" output="log">
    <rosparam file="$(find mocap_optitrack)/config/mocap.yaml" command="load" />
  </node>
</launch>
